frontend/Stats/NumberStats.svelte

```diff
@@ -4,6 +4,7 @@
     import { titleCase, subUnderscores } from "../Common/TextUtils";
     import { Collapse } from "bootstrap";
     import { createEventDispatcher } from "svelte";
+    import { faChevronRight, faDotCircle } from "@fortawesome/free-solid-svg-icons";
```

Comment:
maybe something like
`<i class="fa-regular fa-object-group"></i>`

would be better ?

the cheveron isn't exactly that clear in it's meaning